std/stdio.d

```diff
@@ -17,6 +17,7 @@ Authors:   $(WEB digitalmars.com, Walter Bright),
 module std.stdio;
 
 public import core.stdc.stdio;
+public import std.exception : ErrnoException;
```

Comment:
@DmitryOlshansky  I'm not sure what you mean: the documentation nowhere says that `ErrnoException` is not thrown on Windows. https://github.com/D-Programming-Language/phobos/blob/master/std/stdio.d#L393 uses `errnoEnforce` without any `version(Windows) else` constructs. So why do you say "Windows is not errno-based"?